src/main/java/de/taimos/gpsd4java/backend/SocketThread.java

```diff
@@ -64,7 +64,6 @@ public SocketThread(final BufferedReader reader, final GPSdEndpoint endpoint, fi
 		this.endpoint = endpoint;
 		this.resultParser = resultParser;
 		
-		this.setDaemon(true);
```

Comment:
I think that would be a great idea. Please leave the old constructor with the current behavior as default, too.